docs/src/pages/[platform]/components/input/index.page.mdx

```diff
@@ -27,7 +29,11 @@ export async function getStaticProps() {
 <PageTabLayout 
   tabComponents={[{
     title: "Documentation", 
-    children: <Fragment>{({ platform }) => import(`./${platform}.mdx`)}</Fragment>
+    children: <>
+      <InlineFilter filters={["react"]}>
```

Comment:
Is this necessary since we don't have anything but React here? I don't think Next would generate pages based on the static paths?

Reply:
Nope the `InlineFilter` isn't needed here because there is just the single supported framework.  I've removed it and am just returning the react page but once these components support additional frameworks we'll need to add the `<InlineFilter>` to only display the content for the current framework.